src/main/deltachat/index.js

```diff
@@ -61,6 +61,7 @@ class DeltaChatController extends EventEmitter {
    */
   sendToRenderer (eventType, payload) {
     log.debug('sendToRenderer: ' + eventType)
+    windows.main.send('ALL', eventType, payload)
```

Comment:
why send an all event? ... Oh its for debugging.. Can you check if the loglevel is debug before sending it? Or does it have no performance impact at all?

Reply:
Don't think it will affect performance in a meaningful way